cleanlab/datalab/adapter/imagelab.py

```diff
@@ -0,0 +1,196 @@
+"""Module for adapting Imagelab to the Datalab API.
+
+It uses the Adapter pattern so that Imagelab becomes compatible
+with the DataIssues, IssueFinder, Reporter classes.
```

Comment:
Is this something users would ever care about?

For developer notes, I would explicitly write: 
**Note for developers:** ... 

and assume the documentation is otherwise intended for users.